app/src/main/java/app/gamenative/ui/screen/library/appscreen/GOGAppScreen.kt

```diff
@@ -0,0 +1,668 @@
+     */
+    @Composable
+    override fun AdditionalDialogs(
+        libraryItem: LibraryItem,
+        onDismiss: () -> Unit,
+        onEditContainer: () -> Unit,
+        onBack: () -> Unit,
+    ) {
+        Timber.tag(TAG).d("AdditionalDialogs: composing for appId=${libraryItem.appId}")
+        val context = LocalContext.current
+
+        // Monitor uninstall dialog state
+        var showUninstallDialog by remember { mutableStateOf(shouldShowUninstallDialog(libraryItem.appId)) }
+
+        LaunchedEffect(libraryItem.appId) {
+            snapshotFlow { shouldShowUninstallDialog(libraryItem.appId) }
+                .collect { shouldShow ->
+                    showUninstallDialog = shouldShow
+                }
+        }
+
+        // Monitor install dialog state
+        var showInstallDialog by remember { mutableStateOf(shouldShowInstallDialog(libraryItem.appId)) }
+
+        LaunchedEffect(libraryItem.appId) {
+            snapshotFlow { shouldShowInstallDialog(libraryItem.appId) }
+                .collect { shouldShow ->
+                    showInstallDialog = shouldShow
+                }
+        }
+        // Show install confirmation dialog
+        if (showInstallDialog) {
+            // GOGService expects numeric gameId
+            val gameId = libraryItem.gameId.toString()
+            val gogGame = remember(gameId) {
+                GOGService.getGOGGameOf(gameId)
+            }
+
+            val downloadSizeGB = (gogGame?.downloadSize ?: 0L) / 1_000_000_000.0
+            val sizeText = if (downloadSizeGB > 0) {
+                String.format(Locale.US, "%.2f GB", downloadSizeGB)
+            } else {
+                "Unknown size"
+            }
+
+            AlertDialog(
+                onDismissRequest = {
+                    hideInstallDialog(libraryItem.appId)
+                },
+                title = { Text(stringResource(R.string.gog_install_game_title)) },
+                text = {
+                    Text(
+                        text = stringResource(
+                            R.string.gog_install_confirmation_message,
+                            gogGame?.title ?: libraryItem.name,
+                            sizeText,
+                        ),
+                    )
+                },
+                confirmButton = {
+                    TextButton(
+                        onClick = {
+                            hideInstallDialog(libraryItem.appId)
+                            performDownload(context, libraryItem) {}
+                        },
+                    ) {
+                        Text(stringResource(R.string.download))
+                    }
+                },
+                dismissButton = {
+                    TextButton(
+                        onClick = {
+                            hideInstallDialog(libraryItem.appId)
+                        },
+                    ) {
+                        Text(stringResource(R.string.cancel))
+                    }
```

Comment:
<img width="537" height="310" alt="image" src="https://github.com/user-attachments/assets/67690a80-917d-4c05-beff-36b71b761787" />

Why not use something like `showInstallDialog` from SteamAppScreen? could even move that to BaseAppScreen or something. I know the goal is to get GOG in, but I think we should make these small things consistent before we release.